<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3.1 Gérer les inscriptions à des épreuves</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QComboBox" name="numEp_comboBox3"/>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="num_comboBox3"/>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="boutonModifier">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="numEp_comboBox2"/>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="num_comboBox2">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="boutonSupprimer">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Désinscrire</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="boutonInscrire">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Inscrire</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="numEp_comboBox1"/>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="num_comboBox1"/>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>num</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>numEp</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_editeurInscriptions">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boutonInscrire</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ajouterInscription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonModifier</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>modifierInscription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boutonSupprimer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>supprimerInscription()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num_comboBox1</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshNumComboBox1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num_comboBox2</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshNumComboBox2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numEp_comboBox3</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshNumEpComboBox3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numEp_comboBox2</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshNumEpComboBox2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>num_comboBox3</sender>
   <signal>activated(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>refreshNumComboBox3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ajouterInscription()</slot>
  <slot>supprimerInscription()</slot>
  <slot>modifierInscription()</slot>
  <slot>refreshNumComboBox1()</slot>
  <slot>refreshNumComboBox2()</slot>
  <slot>refreshNumComboBox3()</slot>
  <slot>refreshNumEpComboBox3()</slot>
  <slot>refreshNumEpComboBox2()</slot>
 </slots>
</ui>
